doc/quick_start_guide/ffd/ffd.rst

```diff
@@ -20,14 +20,15 @@ Example designs
 ***************
 
 Demonstration
-===============================
+=============
 
-This is the far-field voice local command (FFD) example design with Sensory TrulyHandsfree™ (THF) speech recognition and local dictionary.
+This is the far-field voice local command (FFD) example design. Two examples are provided: both examples include speech recognition and a local dictionary. One example uses the Sensory TrulyHandsfree™ (THF) libraries, and the other one uses the Cyberon DSPotter™ libraries.
 
 When a wakeword phrase is detected followed by a command phrase, the application will output an audio response and a discrete message over |I2C| and UART.
 
+Sensory's THF and Cyberon's DSpotter™ libraries ship with an expiring development license. The Sensory one will suspend recognition after 11.4 hours or 107 recognition events, and the Cyberon one will suspend recognition after 100 recognition events. Ater the maximum number of recognitions is reached, a device reset is required to resume normal operation.To perform a reset, either power cycle the device or press the SW2 button.
```

Comment:
Same comments as in the overview.rst file.  `Ater` should be `After` and `operation.To` should be `operation. To`.

Reply:
Fixed.